### DISCUSSION TITLE
The beta-10 version of this package broke 2.0.0-beta.3 of Azure.AI.OpenAI, resulting in 404 when calling audio endpoint

### DISCUSSION THREAD

**Discussion by grankko** [openai/openai-dotnet#194] [General]
I filed an issue here https://github.com/Azure/azure-sdk-for-net/issues/45706. Not sure if that was correct or if it should have been filed in this repo.

**Comment by joseharriaga** [comment]
Filing this issue on the `azure-sdk-for-net` repo is correct. Thank you, @grankko ! An update will be released soon to fix the Azure.AI.OpenAI library, but in the meantime, using version 2.0.0-beta.9 of the OpenAI library instead of version 2.0.0-beta.10 should unblock you.
